core/src/main/scala/scalaz/Either.scala

```diff
@@ -404,7 +404,7 @@ object \/ extends DisjunctionInstances {
    * For interfacing with deterministic functions that violate the type system
    * by returning `null`, use [[scalaz.Maybe#fromNullable]].
    */
-  def attempt[A, B](f: => B)(err: Exception => A): A \/ B =
+  def attempt[A, B](err: Exception => A)(f: => B): A \/ B =
```

Comment:
I thought that for `attempt` it made more sense to keep the original order.

Reply:
I have strong feelings about removing `Catchable` too, on the positive side.

Reply:
I guess the aesthetics will depend on what is more verbose in any particular case: the action to attempt or error handling?

I just thought that if it's to be called `attempt`, the first argument should be the action to be attempted.

Don't know of any precedent.

Reply:
ok I'll swap the order back and keep `attempt`